napvig:
  rate: 100
  threads: 1
  mod_flow:
    debug:
      enable: true

  napvig:
    step_ahead_size: 0.2
    gradient_step_size: 0.01
    termination_distance: 0.00
    max_iterations: 100
    landscape:
      measure_radius: 0.4
      smooth_radius: 0.001
      precision: 50
      decimation: 3
    debug:
      output_values: false
      output_gradient: false
      output_range:
        min: -1
        max: 1.15
        step: 0.2
      frames_skip: 5